src/components/person/PersonAutocomplete.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTranslation } from 'react-i18next'
+import { Autocomplete, AutocompleteProps, TextField } from '@mui/material'
+import { PersonResponse } from 'gql/person'
+import { usePersonList } from 'common/hooks/person'
+
+export type PersonAutocompleteProps = {
+  onSelect: (person: PersonResponse | null) => void
+  autocompleteProps?: Omit<
+    AutocompleteProps<PersonResponse, undefined, undefined, undefined>,
+    'renderInput' | 'options' | 'getOptionLabel' | 'onChange' | 'loading'
+  >
+  showId: boolean
+}
+export default function PersonAutocomplete({
+  onSelect,
+  showId,
```

Comment:
Maybe we can add a default value to the boolean field `showId` so we don't have to add it as `showId={false}`

```suggestion
  showId?: boolean
}
export default function PersonAutocomplete({
  onSelect,
  showId = false,
```

Reply:
Done in [95fedb9](https://github.com/podkrepi-bg/frontend/pull/728/commits/95fedb9d7a1c818ab2b7b06e662d78ad23679d4e)